omnipaxos_core/tests/test_config.rs

```diff
@@ -13,6 +13,7 @@ pub struct TestConfig {
     pub num_proposals: u64,
     pub num_elections: u64,
     pub gc_idx: u64,
+    pub storage_type: String,
```

Comment:
Created the StorageTypeSelector enum for deciding storage type based on the hocon file. It will pattern match for either "Memory", "Storage" or panic depending on the input.